### DISCUSSION TITLE
Building MXNet Engine for Android

### DISCUSSION THREAD

**Discussion by LewsTherin511** [apache/incubator-mxnet#19985] [Q&A]
Hi there, I need to deploy some GluonCV models on Android.

I've been suggested to use DJL, a Java high-level library that allows to deploy models created with different frameworks, including MXNet. As I understand, DJL can load engines from different frameworks...however, the MXNet engine for Android is not officially provided.
In the answers to [this question](https://stackoverflow.com/questions/66439247/running-gluoncv-object-detection-model-on-android/66464079#66464079), they suggest to build the MXNet engine for Android and to include it in the Android project, and they pointed to [this link](https://github.com/apache/incubator-mxnet/blob/01387a20c3e54ce859e3849d541d51cf4b428358/ci/docker/runtime_functions.sh#L217-L256).

I ran the

    ./build.py -p armv7

script as indicated in the link, and it produces a Docker image with MXNet installed.
However, I don't really understand how to use this. Can I somehow...export it, and load it in the Android project?

**Comment by leezu** [comment]
You can copy the libmxnet.so obtained via the `build.py` command to your Android project.
